.changeset/warm-sloths-tickle.md

```diff
@@ -0,0 +1,7 @@
+---
+'@aws-amplify/backend-function': patch
+'@aws-amplify/backend-data': patch
+'@aws-amplify/platform-core': patch
+---
+
+Lambda client env var name issue
```

Comment:
This is going to changelogs, can this be better?

Reply:
You're right. This can be improved. Made a revision.